hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/fs/TestFileUtil.java

```diff
@@ -160,7 +162,7 @@ public void setup() throws IOException {
 
     new File(del, FILE).createNewFile();
     File tmpFile = new File(tmp, FILE);
-    tmpFile.createNewFile();
+    assertTrue(tmpFile.createNewFile());
```

Comment:
there's enough use of the same file operations (createNewFile, mkdirs, delete, exists..) that they could be factored out into helper methods which include the assertion the return value *and* meaningful exception error messages

Reply:
Done.